src/inputs/Multi.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="form__multi">
+        <span>{{ this.label }}</span>
+        <div v-for="item in inputValue" :key="item.key"> 
+            <component
+                :value="item.value"
+                :identifier="item.key"
+                :isMulti=true
+                :is="itemType"
+                v-bind="item.props"
+                :index="item.index"
+                v-on:onChange="change"
+                :name="item.name" />
+            <button @click="e => removeInput(e, item.key)"/>
+        </div>
+        <input type="submit" value="Add New" v-on:click="e => addNew(e)">
+        <field-error :errors="this.validationErrors" />
+    </div>
+</template>
+
+<script>
+    import Vue from 'vue';
+    import { SuperText } from '../';
+    import { validationMixin } from '../validationHelper';
+    import { formSchemaMixin } from '../schemaHelper';
+    import _ from 'lodash';
+
+    export default {
+        name: 'SuperMulti',
+        mixins: [validationMixin, formSchemaMixin],
+        mounted() {            
+            this.inputValue = this.inputValue ? this.inputValue.map((val, i) => 
+                this.inputItemFormatter(val, i)
+            ) : [];
+        },
+        computed: {
+            defaultValue() {
+                return ['df', 'nbjh'];
+            },
+            itemType() {
+                return SuperText;
+            }
+        },
+        methods: {
+            change(name, value) {                
+                let newItems = this.inputValue.map(item =>
+                    item.name == name ? item['value'] = value: item['value']
```

Comment:
this is not how a ternary works.
It should look like this:
```javascript
const someVar = condition ? 'conditional value' : 'fallback value';
```
Does the above even work? did you test it?